### DISCUSSION TITLE
More labels for the `build_info` metric

### DISCUSSION THREAD

**Discussion by emschwartz** [autometrics-dev/autometrics-shared#38] [Ideas]
If you have multiple Autometrics-instrumented systems putting metrics into the same Prometheus, it might be useful to have a couple more labels on the `build_info` metric.

I'm thinking along the lines of:
- `language` - `rust`, `python`, etc
- `autometrics_version` - the version of the library being used (although, we aren't versioning the libraries together so this kind of only has meaning together with the `language`)

Anyone have other ideas?

**Comment by emschwartz** [comment]
One very small question: should it be `autometrics.version` for the OpenTelemetry metrics? We might want to have others under that "namespace"

**Reply by mies** [reply]
Would it be possible to have the *exact* python or rust version etc?

**Reply by mies** [reply]
In this case I mean the version of the programming language / runtime version

**Reply by emschwartz** [reply]
We could include the `language.version` as well. What would we do with that info?

**Comment by emschwartz** [comment]
For the language and language version, should we use `langauge` and `language.version` or the OpenTelemetry Semantic Conventions (which are listed as experimental) of [`process.runtime.name` and `process.runtime.version`](https://github.com/open-telemetry/semantic-conventions/blob/c6efa4b2b44f9950b332a46311834e23b85bf979/docs/resource/process.md#process-runtimes)?

**Reply by laulauland** [reply]
Working on a library for a language that now has competing runtimes 🙃 my vote would be for `process.runtime` as it's a bit more specific.

**Reply by emschwartz** [reply]
Great point!
